index.html

```diff
@@ -59,13 +60,12 @@
 
   </div>
 
-  <script src="//cdn.pypyjs.org/pypy.js/0.3.1/lib/Promise.min.js" type="text/javascript"></script>
-  <script src="//cdn.pypyjs.org/pypy.js/0.3.1/lib/FunctionPromise.js" type="text/javascript"></script>
-  <script src="//cdn.pypyjs.org/pypy.js/0.3.1/lib/pypy.js" type="text/javascript"></script>
+  <script src="pypyjs-release/lib/Promise.min.js" type="text/javascript" charset="utf-8"></script>
+  <script src="pypyjs-release/lib/FunctionPromise.js" type="text/javascript" charset="utf-8"></script>
+  <script src="pypyjs-release/lib/pypy.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
I'm not sure whether it makes sense to use the git submodules like this.  IMHO they should only contain released versions of the pypy.js build, so anything in the release repo will also be available via CDN.

I'm happy to try it out though.  What do you see as advantage of using the submodule versus loading from CDN?  Easier local development and experimentation?